core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -6653,13 +6654,44 @@ private void checkLiteral2(String expression, String expected) {
         .withBigQuery().ok(expected);
   }
 
+  @Test void testPivotWithSqlShuttle() {
```

Comment:
I'm wondering if this is the best place for this. Another place would be `SqlParserTest`, there is a similar issue CALCITE-4406 which you can take a look.

Reply:
Agree, thanks, I moved the test there.